flink-connector/src/main/java/com/datastrato/gravitino/flink/connector/catalog/BaseCatalog.java

```diff
@@ -79,10 +80,12 @@
  */
 public abstract class BaseCatalog extends AbstractCatalog {
   private final PropertiesConverter propertiesConverter;
+  private final TransformConverter transformConverter;
```

Comment:
does it will transform distribution , sortorders? if not,  `PartitionConverter` may better